pymc3/__init__.py

```diff
@@ -34,3 +34,21 @@
     _log.setLevel(logging.INFO)
     handler = logging.StreamHandler()
     _log.addHandler(handler)
+
+def tqdm(*args, **kwargs):
+    try:
+        from tqdm import tqdm_notebook
+        return tqdm_notebook(*args, **kwargs)
+    except:
```

Comment:
Yes, that would good.  

For reference here's the exception it catches for me:

    AttributeError                            Traceback (most recent call last)
    <ipython-input-3-f951a61dc95b> in <module>()
    ----> 1 for i in tqdm_notebook(range(10)):
          2     time.sleep(0.1)
          3
 
    /home/bill/anaconda3/lib/python3.5/site-packages/tqdm/__init__.py in tqdm_notebook(*args, **kwargs)
         17     """See tqdm._tqdm_notebook.tqdm_notebook for full documentation"""
         18     from ._tqdm_notebook import tqdm_notebook as _tqdm_notebook
    ---> 19     return _tqdm_notebook(*args, **kwargs)
         20
         21
 
    /home/bill/anaconda3/lib/python3.5/site-packages/tqdm/_tqdm_notebook.py in __init__(self, *args, **kwargs)
        176         # self.sp('', close=True)
        177         # Replace with IPython progress bar display (with correct total)
    --> 178         self.sp = self.status_printer(self.fp, self.total, self.desc)
        179         self.desc = None  # trick to place description before the bar
        180
 
    /home/bill/anaconda3/lib/python3.5/site-packages/tqdm/_tqdm_notebook.py in status_printer(_, total, desc)
        104         ptext = HTML()
        105         # Only way to place text to the right of the bar is to use a container
    --> 106         container = HBox(children=[pbar, ptext])
        107         display(container)
        108
 
    /home/bill/anaconda3/lib/python3.5/site-packages/ipywidgets/widgets/widget_box.py in HBox(*pargs, **kwargs)
         91     """Displays multiple widgets horizontally using the flexible box model."""
         92     box = Box(*pargs, **kwargs)
    ---> 93     box.layout.display = 'flex'
         94     box.layout.align_items = 'stretch'
         95     return box
 
    /home/bill/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in __set__(self, obj, value)
        581             raise TraitError('The "%s" trait is read-only.' % self.name)
        582         else:
    --> 583             self.set(obj, value)
        584
        585     def _validate(self, obj, value):
 
    /home/bill/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in set(self, obj, value)
        570             # we explicitly compare silent to True just in case the equality
        571             # comparison above returns something other than True/False
    --> 572             obj._notify_trait(self.name, old_value, new_value)
        573
        574     def __set__(self, obj, value):
 
    /home/bill/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in _notify_trait(self, name, old_value, new_value)
       1135             new=new_value,
       1136             owner=self,
    -> 1137             type='change',
       1138         ))
       1139
 
    /home/bill/anaconda3/lib/python3.5/site-packages/ipywidgets/widgets/widget.py in notify_change(self, change)
        365             if self._should_send_property(name, change['new']):
        366                 # Send new state to front-end
    --> 367                 self.send_state(key=name)
        368         LoggingConfigurable.notify_change(self, change)
        369
 
    /home/bill/anaconda3/lib/python3.5/site-packages/ipywidgets/widgets/widget.py in send_state(self, key)
        261         state, buffer_keys, buffers = self._split_state_buffers(state)
        262         msg = {'method': 'update', 'state': state, 'buffers': buffer_keys}
    --> 263         self._send(msg, buffers=buffers)
        264
        265     def get_state(self, key=None):
 
    /home/bill/anaconda3/lib/python3.5/site-packages/ipywidgets/widgets/widget.py in _send(self, msg, buffers)
        485     def _send(self, msg, buffers=None):
        486         """Sends a message to the model in the front-end."""
    --> 487         self.comm.send(data=msg, buffers=buffers)
        488
        489
 
    /home/bill/anaconda3/lib/python3.5/site-packages/ipykernel/comm/comm.py in send(self, data, metadata, buffers)
        119         """Send a message to the frontend-side version of this comm"""
        120         self._publish_msg('comm_msg',
    --> 121             data=data, metadata=metadata, buffers=buffers,
        122         )
        123
 
    /home/bill/anaconda3/lib/python3.5/site-packages/ipykernel/comm/comm.py in _publish_msg(self, msg_type, data, metadata, buffers, **keys)
         64         metadata = {} if metadata is None else metadata
         65         content = json_clean(dict(data=data, comm_id=self.comm_id, **keys))
    ---> 66         self.kernel.session.send(self.kernel.iopub_socket, msg_type,
         67             content,
         68             metadata=json_clean(metadata),
 
    AttributeError: 'NoneType' object has no attribute 'session'